pkg/backend/updates.go

```diff
@@ -74,6 +74,13 @@ const (
 	GitHubLogin = "github.login"
 	// GitHubRepo is the name of the GitHub repo, if the local git repo's remote origin is hosted on GitHub.com.
 	GitHubRepo = "github.repo"
+
+	// CISystem is the name of the CI system running the pulumi operation.
+	CISystem = "ci.system"
+
+	// CIPRSourceSHA is the SHA of the HEAD commit of a pull request running on CI. This is needed since the CI
+	// server will run at a different, merge commit.
+	CIPRSourceSHA = "ci.pr.sourceSHA"
```

Comment:
Would prefer if this was called `headSHA` instead of `sourceSHA` to be consistent with how most git providers refer to these things.

Reply:
Sounds great, done.